pages/[id].jsx

```diff
@@ -28,6 +30,7 @@ export default function DetailPage({ fetchedTeamMember }) {
         imageUrl={fetchedTeamMember.photo}
         hours={fetchedTeamMember.forecastedHours}
       />
+      <SkillsGrid skilldata={skills} />
     </Layout>
```

Comment:
Have we thought about what the skills list should look like if there are no values? Do we want to show that column at all or use some filler text? What are your thoughts on this?? 

![Screen Shot 2022-05-18 at 12 21 06 PM](https://user-images.githubusercontent.com/49104801/169117087-c604c466-3d98-4e52-b116-df7a219e817a.png)

Reply:
Earlier today I reached out to Merani about this and am waiting to hear back about her thoughts on how filtering or not filtering fits with the design. I thought I put in the PR description a "don't merge till I confirm this" but I looks like I forgot. Take a look at Legolas and you'll see an even clearer picture that you are right, we do need to think about this.

Reply:
After talking to Merani the answer is to not show columns that have nothing in them. I made this happen by adding a filter in getSkills

Reply:
Looks great!

![Screen Shot 2022-05-19 at 8 28 25 AM](https://user-images.githubusercontent.com/49104801/169319762-56fdc05f-292c-44fd-a2ab-9fdfa9712f31.png)